README.md

```diff
@@ -21,7 +24,7 @@ jobs:
       - uses: gleam-lang/setup-erlang@v1.1.2
         with:
           otp-version: 23.2
-      - run: rebar3 eunit
+      - run: erl -eval 'erlang:display({otp_release, erlang:system_info(otp_release)}), halt().' -noshell
```

Comment:
I think this was easier to understand before when it was using rebar. I think it is rare that someone would want to print the version in CI and this much rather obscure code would be comparitively confusing to newcomers.

Reply:
I took that line from the Windows-example below.
Would `erlc` do? I think separating the tools that comes together with ubuntu-latest and what is actually part of this package would be nice for understanding the improvements this package gives over ubuntu-latest, but that might just be my opinion.
Do you want me to change all examples to `erlc` or `rebar3` or something else?

Reply:
Sorry for the slow reply, renovating my home and starting a new job has kept me really busy.

I don't mind what exactly it is but I would like the command to be clear at a glance rather than being some somewhat cryptic Erlang code that needs to be mentally parsed to understand.